app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java

```diff
@@ -80,4 +80,40 @@ public static void onResponse(int responseCode) {
         }
     }
 
+    /**
+     * Last WindowsSetting constructor values. Values are checked for change to reduce log spam.
+     */
+    private static int lastAp, lastAnchorHorizontalPos, lastAnchorVerticalPos;
+    private static boolean lastZ1, lastZ2;
+
+    /**
+     * Injection point.
+     */
+    public static int[] getSubtitleWindowSettingsOverride(int ap, int anchorHorizontalPos, int anchorVerticalPos,
```

Comment:
Please add inline docs to this method, explaining what the parameters are for future reference if possible. It is unclear what `ap` refers to.